src/selectors/profile.js

```diff
@@ -450,3 +451,43 @@ export const getHiddenTrackCount: Selector<HiddenTrackCount> = createSelector(
     return { hidden, total };
   }
 );
+
+export const getRelevantPagesForActiveTab: Selector<
```

Comment:
I don't see a specific test for this function. It's implementing some tricky logic, so I think a test for it would be a good idea.